src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+class Rectangle extends Figure {
+    private int rightSide;
+    private int topSide;
+
+    public Rectangle(String color, int topSide, int rightSide) {
```

Comment:
Instead of passing expressions directly in the constructor, consider creating separate variables for 'topSide' and 'rightSide' and then pass them to the constructor for better code readability.